spyder/widgets/findinfiles.py

```diff
@@ -576,6 +612,10 @@ def activated(self, item):
             filename, lineno, colno = itemdata
             self.parent().edit_goto.emit(filename, lineno, self.search_text)
 
+    def enable_sorting(self, flag):
+        """Enable result sorting after search is complete."""
+        self.sorting_status['status'] = flag

```

Comment:
Ok, then let's refactor this a bit so it's easier to understand. What about

    self.sorting['status'] = 'on'/'off'

?